src/opustrainer/trainer.py

```diff
@@ -726,7 +730,7 @@ def main() -> None:
     state_tracker = StateTracker(args.state or f'{args.config}.state', restore=not args.do_not_resume)
 
     # Make trainer listen to `kill -SIGUSR1 $PID` to print dataset progress
-    signal.signal(signal.SIGUSR1, lambda signum, handler: print_state(trainer.state(), sys.stderr))
```

Comment:
This is purely for when you *manually* trigger a SIGUSR1. Should that still go to the log file, or always to stderr? I think this one should always go to stderr.

Reply:
hmmm... I think it would like to have duplicated in the log and the stderr? The more i think about it, marian duplicates log to stderr and it's useful.